src/app/globals.css

```
@@ -128,7 +128,8 @@
   * {
     @apply border-border outline-ring/50;
   }
+  html,
   body {
-    @apply bg-background text-foreground;
+    @apply bg-background text-foreground w-full max-w-[100vw] overflow-x-hidden;
```

```suggestion
    @apply bg-background text-foreground;
```
pull from dev to get mobile navbar and footer so you don't need to have these, if the issue persists add an overflow-x-hidden to the classname in layout.tsx, not here